datadragon/model.go

```diff
@@ -272,10 +277,20 @@ type Mastery struct {
 
 // ProfileIcon represents a profile icon
 type ProfileIcon struct {
-	ID    int       `json:"id"`
+	ID    Integer   `json:"id"`
```

Comment:
There's a few ProfileIcons where the id is a quoted number.. absurd to see in an API, but this allows us to use it still

Reply:
typical for the Riot API 🤦🏼‍♂️

Reply:
the stdlib also had [json.Number](https://pkg.go.dev/encoding/json#Number) that COULD have worked, but this implementation makes things a bit more streamline IMO

Reply:
I like this implementation better 👍🏼